mail_service/worker_notify_user.py

```diff
@@ -7,24 +7,28 @@
 bs = beanstalkc.Connection(host="172.17.0.1")
 bs.watch("notify_user")
 
+LOGS_DIR = "/srv/pipeline-logs/"
+LOGS_URL_BASE = "https://registry.centos.org/pipeline-logs/"
```

Comment:
I have noticed these constants being declared explicitly in multiple places. Would it not be a good idea to put it in one place and import it everywhere instead?